### ISSUE TITLE
The installer creates a shortcut with an invalid --jdkhome value

### ISSUE THREAD

**Issue by ItsIgnacioPortal** [apache/netbeans#7343]
### Apache NetBeans version

Apache NetBeans 21

### What happened

When installing Netbeans from chocolatey, the package for v21 uses the URL https://dlcdn.apache.org/netbeans/netbeans-installers/21/Apache-NetBeans-21-bin-windows-x64.exe to fetch the installer. That installer runs correctly and installs Netbeans, BUT the Windows shortcut it generates on the startup folder has in its `Target` value the following string:

```
C:\ProgramData\chocolatey\lib\apache-netbeans.portable\App\netbeans\bin\netbeans64.exe --jdkhome "C:\Program Files\Eclipse Adoptium\jdk-17.0.9.9-hotspot\"
```

Notice the value that is given for the `--jdkhome` parameter. That folder did not exist on my system, which caused Netbeans to always fail to find the correct jdk location automatically. If that parameter wasn't there, netbeans would've used the JDK_HOME system environment variable to find the jdk location, and it would've succeded.

I've verified that its not the chocolatey package the one who adds that line to the shortcut, therefor it must the the installer the one who has that string hard-coded in. At some point, this hard-coded path is added into the installer. I did a github code search and the path wasn't present anywhere in this repository, so my best guess is that it's [the Apache Jenkins build CD](https://ci-builds.apache.org/job/Netbeans/job/netbeans-windows/) the one who's adding the path.

### Language / Project Type / NetBeans Component

N/A

### How to reproduce

1. [Install chocolatey](https://chocolatey.org/install)
2. Run `choco install netbeans` in a powershell window as Administrator.
3. Open netbeans

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 10 Version 22H2 (OS auild 19045.4291)

### JDK

Openjdk 21.0.2

### Apache NetBeans packaging

Third-party package

### Anything else

Information about the third-party chocolatey package is available at https://community.chocolatey.org/packages/NetBeans/21.0.0

### Are you willing to submit a pull request?

No

**Comment by ebarboni** [comment]
No sure to understand. The default installer create a shortcut icon on the desktop with no jdk parameters. The jdk parameter is in the netbeans/conf folder.

**Comment by ItsIgnacioPortal** [comment]
> No sure to understand. The default installer create a shortcut icon on the desktop with no jdk parameters. The jdk parameter is in the netbeans/conf folder. 

Please try to reproduce the bug using the instructions I shared before discarding it as impossible.

**Comment by neilcsmith-net** [comment]
No, please try to replicate without Chocolatey, or report upstream to them.

**Comment by ItsIgnacioPortal** [comment]
> No, please try to replicate without Chocolatey, or report to them first.

As per my original report, I've already verified that this issue is NOT caused by chocolatey. Here are some simpler issue-reproduction instructions that don't involve chocolatey:

## How to reproduce
1. Download the Apache Netbeans Windows installer. Both of the following links host the exact same file (hashes match). You may download the installer from either of them:
	- https://dlcdn.apache.org/netbeans/netbeans-installers/21/Apache-NetBeans-21-bin-windows-x64.exe
	- https://www.apache.org/dyn/closer.lua/netbeans/netbeans-installers/21/Apache-NetBeans-21-bin-windows-x64.exe
2. Execute the installer
3. Open Netbeans
